internal/local/cmd.go

```diff
@@ -184,8 +184,23 @@ func New(provider k8s.Provider, portHTTP int, opts ...Option) (*Command, error)
 				cmd = exec.Command("open", url)
 			case "windows":
 				cmd = exec.Command("cmd", "/c", "start", url)
+			case "linux":
+				if _, err := exec.LookPath("xdg-open"); err == nil {
+					cmd = exec.Command("xdg-open", url)
+				} else if _, err := exec.LookPath("sensible-browser"); err == nil {
+					cmd = exec.Command("sensible-browser", url)
+				} else if _, err := exec.LookPath("x-www-browser"); err == nil {
+					cmd = exec.Command("x-www-browser", url)
+				} else if _, err := exec.LookPath("gnome-open"); err == nil {
+					cmd = exec.Command("gnome-open", url)
+				} else if _, err := exec.LookPath("open"); err == nil {
+					cmd = exec.Command("open", url)
+				} else {
+					return errors.New("unable to determine executable to launch web-browser")
+				}
+
 			default:
-				cmd = exec.Command("xdg-open", url)
+				return fmt.Errorf("unsupported platform '%s'", runtime.GOOS)
```

Comment:
Can we write a message that says something like, please open HTTP://localhost:<port> in your browser, something that can be clicked would be preferred.

Reply:
This messaging is already handled outside of this function - [here](https://github.com/airbytehq/abctl/pull/22/files#diff-1339aa0df3078ccb2014bd86b5df19d27fa84f7f5193f290a8e2ffcdb010a027R520-R521)